provisioner/template.go

```diff
@@ -96,7 +96,6 @@ func renderTemplate(context *templateContext, file string) (string, error) {
 		"azCount":              azCount,
 		"split":                split,
 		"join":                 sprig.GenericFuncMap()["join"],
-		"eksID":                eksID,
```

Comment:
I didn't find any use of `eksID()` in kubernetes-on-aws. I think we might be able to drop it fyi @mikkeloscar